overlays/default.nix

```diff
@@ -13,22 +13,13 @@
 
   (final: _prev: {
     unstable = import edge-nixpkgs {
-      inherit (final) system config;
+      inherit (final) config;
+      inherit (final.stdenvNoCC.hostPlatform) system;
     };
   })
 
   # Patched and override existing name because of it is not configurable
   (final: prev: {
-    # TODO: Use `services.gnome.gcr-ssh-agent.enable = false` since nixos-25.11
```

Comment:
false? true? 🤔

Reply:
I'm not sure similar not to https://gitlab.gnome.org/GNOME/gnome-online-accounts/-/issues/414, https://gitlab.gnome.org/GNOME/nautilus/-/issues/3811#note_2393048

There is no log https://gitlab.gnome.org/GNOME/gnome-online-accounts/-/wikis/Debugging#logging

Reply:
cosmic-files is also unavailable from another problem: `https://github.com/NixOS/nixpkgs/issues/464392#issuecomment-3578385217`

```console
> cosmic-files
thread 'main' (29785) panicked at /build/cosmic-files-1.0.0-beta.7-vendor/iced_winit-0.14.0-dev/src/program.rs:187:39:
Create event loop: Os(OsError { line: 63, file: "/build/cosmic-files-1.0.0-beta.7-vendor/winit-0.30.5/src/platform_impl/linux/wayland/event_loop/mod.rs", error: NoWaylandLib })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
https://github.com/kachick/dotfiles/pull/1363/commits/6b015d4bf61c19f0149e684ddb97fd9cd4af33a6